pkgs/applications/networking/nextcloud-client/wrapper.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, nextcloud-client, makeWrapper, symlinkJoin, withGnomeKeyring ? false, libgnome-keyring }:
+
+if (!withGnomeKeyring) then nextcloud-client else symlinkJoin {
+  name = "${nextcloud-client.name}-with-gnome-keyring";
+  paths = [ nextcloud-client ];
+  nativeBuildInputs= [ makeWrapper ];
+
+  postBuild = ''
+    wrapProgram "$out/bin/nextcloud" \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libgnome-keyring ]}
+  '';
+
+  inherit (nextcloud-client) meta;
```

Comment:
Shouldn't this meta be different to explicitly say that it has gnome-keyring support?

Reply:
> I considered an altered derivation name to be sufficient, so whenever you rebuild, you'll see that. 
Glanced over that, I think that's fine as well.